oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -287,5 +287,130 @@ int DumpHandler::read(uint32_t offset, uint32_t& length, Response& response,
     return readFile(resDumpDirPath, offset, length, response);
 }
 
+int DumpHandler::fileAckWithMetaData(uint32_t metaDataValue1,
+                                     uint32_t metaDataValue2,
+                                     uint32_t /*metaDataValue3*/,
+                                     uint32_t /*metaDataValue4*/)
+{
+    auto path = findDumpObjPath(fileHandle);
+    uint8_t fileStatus = (uint8_t)metaDataValue2;
+    if (dumpType == PLDM_FILE_TYPE_RESOURCE_DUMP_PARMS)
+    {
+        std::cerr << "Token: " << metaDataValue1 << std::endl;
```

Comment:
if you want to keep can u pls make it as std::cout as its not an error

Reply:
done